### PR TITLE
Update README.md by including email

### PR DESCRIPTION
**Pull request by Bharat123rox**
Make it clear that the link is a redirect to an email and not to any members page or developer team page

# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by thinkingserious** [comment]
This content is redundant, but I encourage you to participate in our [Hacktoberfest celebration](https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/)!

File: README.md [hunk 1]
```diff
@@ -224,7 +224,8 @@ Please see our [troubleshooting guide](https://github.com/sendgrid/sendgrid-pyth
 <a name="about"></a>
 # About
 
-sendgrid-python is guided and supported by the SendGrid [Developer Experience Team](mailto:dx@sendgrid.com).
+sendgrid-python is guided and supported by the SendGrid Developer Experience Team.
```
Comment (thinkingserious) [inline README.md]: Please add a space between these two lines. Thanks!

Comment (Bharat123rox) [reply README.md]: Fixed spacing and alphabetical order

**Comment by thinkingserious** [comment]
Hello @Bharat123rox,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**